File: numero narcisista.js
function narcissistic(numero) {
  
  
  
  let digitos = numero.toString()
  
  let numeroDigitos  = digitos.length
  
  let suma=0
  
  for (let digito of digitos){
     suma +=Math.pow(parseInt(digito), numeroDigitos)
    }
  return suma === numero
    
  }

console.log (narcissistic(153))
console.log (narcissistic(1938))
console.log (narcissistic(9474))
console.log (narcissistic(3000))

File: numero narcisista.py
def narcissistic(value):
    numero = str(value)

    suma = 0

    for i in numero:
        suma += int(i) ** len(numero)
    return suma == value
  


print (narcissistic(153))
print (narcissistic(1938))
print (narcissistic(9474))
print (narcissistic(3000))
